### DISCUSSION TITLE
open firefox,wait 1min and then quit

### DISCUSSION THREAD

**Discussion by juggofdotahp** [termux/termux-app#3592] [Q&A]
sometime show----     JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.
console.error: "SearchSuggestionController found an unexpected string value:" "HTTP request timeout" 

——-------------------------

sometime show-----    JavaScript error: resource://gre/modules/XULStore.sys.mjs, line 60: Error: Can't find profile directory.

(xfce4-terminal:31165): Gtk-WARNING **: 11:00:27.775: Could not load a pixbuf from /org/gtk/libgtk/theme/Adwaita/assets/check-symbolic.svg.
This may indicate that pixbuf loaders or the mime database could not be found.

**Comment by dr1408** [comment]
I have same errors any fix ?
